public/forms/validation/form.ttl

```diff
@@ -0,0 +1,159 @@
+  sh:name "Field name" ;
+  sh:order 1 ;
+  sh:path sh:name ;
+  form:displayType displayTypes:defaultInput ;
+  sh:group ext:formFieldPg .
+
+ext:formNodesL a form:Listing ;
+  form:each ext:formValidationItem ;
+  form:scope ext:formNodesS ;
+  sh:group ext:formFieldPg ;
+  sh:order 3 .
+
+##########################################################
+#  Field with optional validations
+##########################################################
+ext:formValidationItem a form:SubForm;
+  form:includes ext:formNodesNameF ;
+  form:includes ext:formValidationL .
+
+##########################################################
+# Field validations
+##########################################################
+ext:formValidationL a form:Listing ;
+  form:each ext:formValidationValueItem ;
+  form:scope ext:formValidationsS ;
+  form:canAdd true ;
+  form:addLabel "Validation" ;
+  form:canRemove true ;
+  form:removeLabel "Remove section" ;
+  form:createGenerator ext:addValidationGenerator ;
+  sh:group ext:formFieldPg ;
+  sh:order 3 .
+
+ext:formValidationValueItem a form:SubForm ;
+  form:hasFieldGroup ext:validationsFG ;
+  form:includes ext:validationResultMessageF .
+
+ext:validationsFG a form:FieldGroup ;
+  form:hasField fields:validatonSelectorF .
+
+##########################################################
+#  Validation type selector
+##########################################################
+fields:validatonSelectorF a form:Field ;
+  sh:name "Validation type" ;
+  sh:order 2 ;
+  sh:path rdf:type ;
+  form:help """
+   """;
+  form:validations 
+    [ a form:RequiredConstraint ;
+        form:grouping form:Bag ;
+        sh:path rdf:type ;
+        sh:resultMessage "Dit veld is verplicht."@nl
+    ] ;
+  form:displayType displayTypes:conceptSchemeSelector ;
+  form:options """{"conceptScheme":"http://lblod.data.gift/concept-schemes/dde3d2a3-e848-47ea-ba44-0f2e565f04ab","searchEnabled":true}""" ;
+  sh:group ext:formFieldPg ;
+  form:hasConditionalFieldGroup fields:maxLengthValidationCFG .
+
+##########################################################
+#  Validation generator
+##########################################################
+ext:addValidationGenerator a form:Generator ;
+  form:prototype [
+    form:shape [
+      form:grouping "Bag"@en
+    ]
+  ] .
+
+##########################################################
+#  Field Groups per validation type
+##########################################################
+fieldGroups:maxLengthValidationFG a form:FieldGroup ;
+  form:hasField ext:validationMaxLengthF .
+
+##########################################################
+#  Validation type conditions
+##########################################################
+fields:maxLengthValidationCFG a form:ConditionalFieldGroup ;
+  form:conditions
+    [ a form:SingleCodelistValue ;
+        form:grouping form:Bag ;
+        sh:path rdf:type ;
+        form:conceptScheme concept:dde3d2a3-e848-47ea-ba44-0f2e565f04ab ;
```

Comment:
I would give the concept a more descriptive uri instead of a UUID, or is this something that will be used outside of the form-builder as well?

Reply:
I could change it in the meta.ttl file as I only found [results on this ID](https://github.com/search?q=org%3Alblod%20dde3d2a3-e848-47ea-ba44-0f2e565f04ab&type=code) in the `form-builder` itself.

Does this also mean that we will be having  to create a migration for these as they are added to the database in the [backend](https://github.com/lblod/app-form-builder/blob/7e74170f0a26932a5fc58667919a865eb70dc642/config/migrations/20210305123324-add-validations.sparql#L11)?

Reply:
Maybe, I'm not sure why we inserted these in the DB actually. Don't we only need them in the frontend?

Reply:
Than this was a good question :D It actually confused me a bit because when changing the objects in the `meta.ttl` (for example: the `form:required` in meta.ttl  now was a full uri before) file it actually just worked in the frontend so that made me think that it is indeed not needed that it is stored in the database. 

Another thing thing is when starting with the validation form @aatauil  got me quering the database and putting that ttl result in the valdiations meta ttl file. This again confirms that we do not need to write migrations for it?

Ill change the id to something more descriptive :)

Reply:
Changed it to `http://lblod.data.gift/concept-schemes/possibleValidations`